pom.xml

```diff
@@ -48,28 +49,39 @@
         <dependencies>
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
-                <artifactId>bom-2.235.x</artifactId>
-                <version>11</version>
+                <artifactId>bom-2.249.x</artifactId>
+                <version>950.v396cb834de1e</version>
                 <type>pom</type>
                 <scope>import</scope>
             </dependency>
             <dependency>
-                <groupId>com.fasterxml.jackson.core</groupId>
-                <artifactId>jackson-databind</artifactId>
-                <version>2.11.1</version>
+                <!-- fix upper bound -->
+                <groupId>joda-time</groupId>
+                <artifactId>joda-time</artifactId>
+                <version>2.10.2</version>
+            </dependency>
+            <dependency>
+                <groupId>org.powermock</groupId>
+                <artifactId>powermock-module-junit4</artifactId>
+                <version>${powermock.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.powermock</groupId>
+                <artifactId>powermock-api-mockito2</artifactId>
+                <version>${powermock.version}</version>
```

Comment:
We recommend plugins not use Powermock. Is there a reason you readded the dep?

Reply:
The plugin uses it in https://github.com/jenkinsci/aws-global-configuration-plugin/blob/c3ffddb806d4889c71dca05467463a89665edeb7/src/test/java/io/jenkins/plugins/aws/global_configuration/CredentialsAwsGlobalConfigurationMockTest.java#L64, and the parent pom removed it.

Reply:
Yes, because we do not want plugins to use it any more. Please read through the options in https://github.com/jenkinsci/plugin-pom/pull/442.

Reply:
In this case I would be fine with `CredentialsAwsGlobalConfigurationMockTest` being deleted, as it pretty much just asserts obvious parts of the implementation, but does not provide any confidence that AWS configuration works in a realistic scenario (such as against AWS or at least MinIO or OpenStack).

Reply:
Fine, I'll remove that test class then.